### PR TITLE
used the post method for signup

### PR DESCRIPTION
**Pull request by Vaishnavi23032001**
Check the post method using axios in signup page

### CODE REVIEW

File: src/components/LoginForm.jsx [hunk 1]
```diff
@@ -11,6 +12,11 @@ const LoginForm = () => {
     value: "",
     isTouched: false,
   });
+
+  useEffect(() => {
+    document.title = "Login";
```
Comment (TjGanesh) [inline src/components/LoginForm.jsx:17]: change this to the component and not access JS Element

File: src/components/SignUp.jsx [hunk 2]
```diff
@@ -97,26 +103,38 @@ function SignUp() {
...
+      fullName: fullName.value,
+      email: email.value,
+      mobileNumber: mobileNumber.value,
+      username: userName.value,
+      password: password.value,
+      role: role,
+    };
+    
+    axios
+      .post("https://15f6-203-212-221-36.ngrok-free.app/api/users/sign-up", data)
```
Comment (TjGanesh) [inline src/components/SignUp.jsx:118]: use .env file to store the API endpoints

**Review by TjGanesh — CHANGES_REQUESTED** [review]
please review the changes!!

File: src/components/LoginForm.jsx (hunk 1 above)
Comment (Vaishnavi23032001) [reply src/components/LoginForm.jsx:17]: Done

File: src/components/SignUp.jsx (hunk 2 above)
Comment (Vaishnavi23032001) [reply src/components/SignUp.jsx:118]: changes done

**Comment by Vaishnavi23032001** [comment]
fixed changes

**Pull request closed without merging**